website/docs/language/settings/backends/configuration.html.md

```diff
@@ -5,11 +5,14 @@ page_title: "Backend Configuration - Configuration Language"
 
 # Backend Configuration
 
-
 Each Terraform configuration can specify a backend, which defines where
-[state](/docs/language/state/index.html) snapshots are stored. Most non-trivial Terraform
-configurations configure a remote backend so that multiple people can work with the same
-infrastructure.
+[state](/docs/language/state/index.html) snapshots are stored.
+
+[Terraform Cloud integration](/docs/language/settings/configuring-terraform-cloud)
+is another way to store state remotely, and is mutually exclusive with configuring a backend.
```

Comment:
"Terraform Cloud integration" as a proper noun feels weird to me; it seems like this is a place where we could just say "Terraform Cloud"?

Reply:
I had some similar language challenges to contend with in initial drafting of the UI language in a recent PR, and tried to always resolve it by formulating the sentence so that "Terraform Cloud" alone would be either the subject or object of the sentence, with the aim that we seem to always be talking about Terraform Cloud as a whole (of which the CLI integration code is a part) rather than this specific implementation detail of there being a slice of client code in CLI.

With that in mind, I'd offer the following rewording in similar vein to what I was doing in the error messages I added:

> Backend configuration is not needed when you are using Terraform Cloud, because Terraform Cloud manages state automatically for each workspace. Use a `cloud` block instead.

(where "a `cloud` block" is the link to the documentation about that configuration block.)

Reply:
Oh thanks, that's a good suggestion!